engine/src/main/java/org/terasology/engine/module/ModuleManager.java

```diff
@@ -79,16 +79,20 @@ public ModuleManager(String masterServerAddress, List<Class<?>> classesOnClasspa
 
         loadModulesFromApplicationPath(pathManager);
 
+        ensureModulesDependOnEngine(engineModule);
```

Comment:
I wondered why this is important, and found https://github.com/MovingBlocks/Terasology/issues/1450

It's not a worry that the `engine` module wouldn't be loaded without it. It's about ordering: some things run in an order derived from the dependency tree, and we want to make sure engine is at the root of it.

Reply:
```suggestion
        // I wondered why this is important, and found #1450
        // It's not a worry that the engine module wouldn't be loaded without it. 
        // It's about ordering: some things run in an order derived from the dependency 
        // tree, and we want to make sure engine is at the root of it.
        ensureModulesDependOnEngine(engineModule);
```

Reply:
Added this as comment to `ensureModulesDependOnEngine` for later reference.